addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -1,44 +1,64 @@
-<div class="overlay">
-  <div class="modal">
+{{#ember-remodal
+  forService=true
+  closeOnOutsideClick=false
+  onOpen=(action 'modalOpen')
+  onClose=(action 'modalClose')
+  onConfirm=(action 'confirm')
+  closeOnEscape=false
+  disableNativeClose=true as |modal|
+}}
+  {{yield}}
+  {{#modal.open}}
```

Comment:
The yield wouldnt work properly if it was wrapping the modal.open block for some reason. Also if the block isnt defined in the template, nothing shows anyway. The #modal.open is just a block that connects whatever it contains to the onOpen action on click.